### PR TITLE
Makes CallSignature.call_signature()

### PR DESCRIPTION
**Pull request by Alexey-T**
It is same result as in docstring, ie method .doc, ie property. So it is with "self" (for methods) -i dont like it, but no other way, easy.

### CODE REVIEW

**Comment by davidhalter** [comment]
Tests? I'm not sure about the naming yet, but agree that we should have something like this.

**Comment by Alexey-T** [comment]
Tests... well, tested on working Cudatext plugin. not sure how to make tests

**Comment by Alexey-T** [comment]
Added one test

**Comment by Alexey-T** [comment]
Was wrong text, now its OK

File: test/test_api/test_call_signatures.py [hunk 1]
```diff
@@ -21,6 +21,15 @@ def assert_signature(source, expected_name, expected_index=0, line=None, column=
         return signatures[0]
 
 
+def assert_signature_string(source, line, column, expected_string):
+    signatures = Script(source, line, column).call_signatures()
+
+    assert len(signatures) <= 1
+
+    if signatures:
+        assert signatures[0].call_signature == expected_string
```
Comment (davidhalter) [inline test/test_api/test_call_signatures.py]: There should be an 

    else:
        assert expected_string is None

there.

The test below is probably running into this.

**Comment by Alexey-T** [comment]
So the test is fixed, you agree?

**Comment by Alexey-T** [comment]
it is fixed.

**Comment by davidhalter** [comment]
Sorry for taking so long on this, but I didn't like what we came up with as the API together. It took a long time to develop another one, because the old internals didn't support what we were doing at the time.

Now there's `[signature.to_string() for signature in definition.get_signatures()]. Thanks for your patience.

**Pull request closed without merging**